src/lava/lib/peripherals/dvs/prophesee_rawreader.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
Is this file still needed? You seem to have ProcessModels for both the EventsIterator and RawReader implementation in the other file.